### PR TITLE
Implement ParquetFileAnalyzer for hadoop-dataset-extractor-standalone

### PR DESCRIPTION
**Pull request by tsukaby**
I want to get metadata on parquet files in HDFS. The parquet core library is difficult to use, therefore I used the parquet-avro library.

I succeeded except getting codec. Is this OK?

Thank you!

### CODE REVIEW

File: hadoop-dataset-extractor-standalone/build.gradle [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies {
   extraLibs externalDependency.avro_mapred
   extraLibs externalDependency.hive_exec
   extraLibs externalDependency.pig
+  extraLibs 'org.apache.parquet:parquet-avro:1.8.1'
```
Comment (mars-lan) [inline hadoop-dataset-extractor-standalone/build.gradle]: Please move this to externalDependency here: https://github.com/linkedin/WhereHows/blob/master/build.gradle#L85

**Comment by mars-lan** [comment]
This should fix https://github.com/linkedin/WhereHows/issues/452

File: hadoop-dataset-extractor-standalone/build.gradle [hunk 2]
```diff
@@ -32,6 +33,7 @@ dependencies {
   compile externalDependency.http_client
   compile externalDependency.http_core
   compile externalDependency.logback
+  compile 'org.apache.parquet:parquet-avro:1.8.1'
```
Comment (mars-lan) [inline hadoop-dataset-extractor-standalone/build.gradle]: Mind to change this line as well? Thanks!

**Comment by tsukaby** [comment]
Thank you! 👍

**Pull request merged**